From 3e0e5636effabe150374f2a212974f1e954be5b2 Mon Sep 17 00:00:00 2001
From: Peter Ujfalusi <peter.ujfalusi@ti.com>
Date: Tue, 28 Jun 2011 10:16:55 +0000
Subject: [PATCH 018/149] omap: board-omap3evm: Fix compilation error

Fix compilation error introduced with 786b01a8c1db0c0decca55d660a2a3ebd7cfb26b
(cleanup regulator supply definitions in mach-omap2).

Signed-off-by: Peter Ujfalusi <peter.ujfalusi@ti.com>
[tony@atomide.com: updated comments]
Signed-off-by: Tony Lindgren <tony@atomide.com>
---
 arch/arm/mach-omap2/board-omap3evm.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/mach-omap2/board-omap3evm.c b/arch/arm/mach-omap2/board-omap3evm.c
index 6f957ed..57bce0f 100644
--- a/arch/arm/mach-omap2/board-omap3evm.c
+++ b/arch/arm/mach-omap2/board-omap3evm.c
@@ -510,7 +510,7 @@ static struct regulator_init_data omap3evm_vio = {
 #define OMAP3EVM_WLAN_IRQ_GPIO		(149)
 
 static struct regulator_consumer_supply omap3evm_vmmc2_supply[] = {
-	REGULATOR_SUPPLY("vmmc", "omap_hsmmc.1");
+	REGULATOR_SUPPLY("vmmc", "omap_hsmmc.1"),
 };
 
 /* VMMC2 for driving the WL12xx module */
@@ -518,7 +518,7 @@ static struct regulator_init_data omap3evm_vmmc2 = {
 	.constraints = {
 		.valid_ops_mask	= REGULATOR_CHANGE_STATUS,
 	},
-	.num_consumer_supplies	= ARRAY_SIZE(omap3evm_vmmc2_supply);,
+	.num_consumer_supplies	= ARRAY_SIZE(omap3evm_vmmc2_supply),
 	.consumer_supplies	= omap3evm_vmmc2_supply,
 };
 
-- 
1.7.2.5

